server/service/src/item_stats.rs

```diff
@@ -151,8 +153,9 @@ impl ItemStats {
             available_stock_on_hand: row.available_stock_on_hand,
             item_id: requisition_line.item_row.id.clone(),
             item_name: requisition_line.item_row.name.clone(),
-            // TODO: Implement total consumption
+            // TODO: Implement total consumption & total_stock_on_hand
             total_consumption: 0.0,
+            total_stock_on_hand: 0.0,
```

Comment:
This is a bit gross. Currently unused in the context where it is defaulted to 0.

Maybe worth separating into a different struct?

Reply:
I am tempted to make another query just for the report (and not add total_stock_on_hand, but I think since total_consumption is also set to zero here, I think it should be ok, thanks